blockchain/contracts/MainContract.sol

```diff
@@ -205,15 +203,19 @@ contract NFTtheWorld {
             nftOwnerships[_dropHash][nftIndex].owner.transfer(
                 nftOwnerships[_dropHash][nftIndex].weiPrice
             );
+            nftReservations[msg.sender][_dropHash] --;
+            dropData[_dropHash].reservedCount --;
```

Comment:
Please don't change that, even though the NFTs are now bought, they still were reserved, that didn't change

Reply:
Both counts?
What should be done to the counts in revertTimedoutReservations() ?

Reply:
Sorry, only the dropData one. Because if there are NFTs left, a new drop is going to be created anyways :) So we don't need to change the reservedCount data of this one.